.travis.yml

```diff
@@ -5,11 +5,11 @@ dist: trusty
 sudo: false
 
 php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
+  - '5.5'
+  - '5.6'
+  - '7.0'
+  - '7.1'
+  - '7.2'
```

Comment:
Actually I don't know why do we need these changes. As I know YAML works fine with string w/o spaces and they do not need escaping. Could you elaborate on this?

Reply:
These are not treated as strings, without the quotes, and are in fact treated as floats. Type juggling allows this to sometimes work as expected, but I don't like to rely on it, given the way floats handle rounding and often even formatting (I've encountered errors where `2.2` becomes `'2.200000'`, for example, which of course doesn't match as expected for a version number, which isn't a float at all). I recognize it's a preference thing, though, so I can revert that if we'd prefer to just leave it alone.